src/frontends/onnx/frontend/src/op/scatter_elements.cpp

```diff
@@ -17,9 +18,14 @@ OutputVector scatter_elements(const Node& node) {
     const auto data = node.get_ng_inputs().at(0);
     const auto indices = node.get_ng_inputs().at(1);
     const auto updates = node.get_ng_inputs().at(2);
-
     const auto axis_node = node.get_attribute_as_constant<std::int64_t>("axis", 0);
-
+    if (node.has_attribute("reduction")) {
+        const auto reduction = node.get_attribute_value<std::string>("reduction", "none");
+        CHECK_VALID_NODE(node,
+                         reduction == "none",
+                         "Unsupported type of attribute: `reduction`. Only `none` is supported",
+                         reduction);
```

Comment:
I think the "type" word can be misleading in this case, and the actual `reduction` value will be printed without context at the end of the message.
```suggestion
                         reduction == "none",
                         "Unsupported value of attribute: `reduction`. Only `none` is supported, got: ",
                         reduction);
```

Reply:
Changed 😊